Kernel/API/SyscallString.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
This needs a license header, as the linter says :)

Reply:
Ack! :)